src/pages/RegistrationPage/RegistrationPage.js

```diff
@@ -12,33 +13,41 @@ import Card from '@material-ui/core/Card';
 import { CardHeader, CardContent, CardActions } from '@material-ui/core';
 import Button from '@material-ui/core/Button';
 import Logo from '../../components/Logo/Logo';
+import { Alert } from '@material-ui/lab';
 
 import { useStyles } from './RegistrationPage.styles';
 
 const RegistrationPage = () => {
-  const [email, setEmail] = useState({ input: {value: ''}, isLocked: false });
```

Comment:
Maybe email field should always be locked, because if you will not get the email, then it will redirect to error page.